docs/index.md

```diff
@@ -1593,6 +1593,12 @@ repositories:
     url: https://<MyRegistry>.azurecr.io/helm/v1/repo
 ```
 
+### Java Runner
+The [jhelmfile](https://github.com/paichinger/jhelmfile) library can be used to run `helmfile` using java. It's a wrapper library which can be configured to run helmfile via an installed binary or using docker. It provides conviniece to create helmfile commands programmatically, run them  and map their output to java objects which can then be used for further processing.  
```

Comment:
Similarly to https://github.com/helmfile/helmfile/pull/1002/files#r1317958140, we'd better replace this with a simple link to some section in your project README!